docs/mobile-apps/features/audio-capture.md

```diff
@@ -70,23 +70,27 @@ If your apps's manifest.xml file defines `android:allowAudioPlaybackCapture="fal
 
 :::
 
-## Using TalkBack on Android Private Devices
+## Enabling TalkBack/Voiceover on your Devices
 
-To utilize TalkBack on your Android private device, make sure you have a device with TalkBack enabled (Android 10 and above, Google Pixel or Samsung devices). Follow these steps:
+To utilize TalkBack on your iOS/Android devices, make sure you have access to supported devices, We support iOS 13 and above, and Android 10 and above. 
```

Comment:
... utilize VoiceOver/TalkBack on your ...
Maybe keep the order for Android or iOS first the same as in the headline

Reply:
Should we `Audio Accessibility` term to cover both `VoiceOver/TalkBack`.

Reply:
it should cover. but i wanna make it more easily discoverable. i wanna call out everywhere Talkback/Voiceover